We do not need to check docs for this project.

--- greenlet-3.5.1/tox.ini.orig
+++ greenlet-3.5.1/tox.ini
@@ -6,11 +6,9 @@
 commands =
     python -c 'import greenlet._greenlet as G; assert G.GREENLET_USE_STANDARD_THREADING'
     python -m unittest discover -v greenlet.tests
-    sphinx-build -b doctest -d docs/_build/doctrees-{envname} docs docs/_build/doctest-{envname}
 sitepackages = False
 extras =
     test
-    docs
 
 
 
